docs/upgrade-pro.md

```diff
@@ -0,0 +1,61 @@
+# Upgrade to Percona Server for MySQL Pro
+
+Are you a Percona Customer already and are you ready to enjoy all the [benefits of Percona Server for MySQL Pro](../psmysql-pro.md)? 
+
+This document provides instructions on how you can upgrade from Percona Server for MySQL Basic to Percona Server for MySQL Pro.
```

Comment:
Are we calling it Percona Server for MySQL Basic or just Percona Server for MySQL
I'd ask Val to clarify.